src/server/util/localstack.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable import/prefer-default-export */
+import { injectable } from 'inversify'
+import AWS from 'aws-sdk'
+
+import { FileVisibility, S3Interface } from './aws'
+import { logger } from '../config'
+
+const BUCKET_NAME = process.env.AWS_S3_BUCKET || 'local-bucket'
+const BUCKET_ENDPOINT = 'http://localstack:4572'
+const ACCESS_ENPOINT = 'http://localhost:4572'
```

Comment:
Few points: 

1) Will there be any instances where bucket_endpoint deviates in value from access_endpoint? if not, they can be in the same variable maybe

2) Feels like some of these can go into our config.ts, such as the BUCKET_NAME. Actually, will there be issues if the bucket_name just uses what's defined in the S3_BUCKET env var (i.e. file-staging.go.gov.sg)?

Reply:
1. The bucket endpoint is different from the access endpoint. The bucket endpoint is the endpoint for our containerised app to communicate with our localstack.

2. Good point there. But I felt it will be good to differentiate the actual staging bucket name from localstack bucket name. Point taken for the part on config.ts, I will be shifting relevant localstack variables there.